docs/domains/Domain.md

```diff
@@ -157,6 +158,16 @@ ServerPod describes the configuration for a Kubernetes pod for a server.
 | periodSeconds | number | The number of seconds between checks |
 | timeoutSeconds | number | The number of seconds with no response that indicates a failure |
 
+### Shutdown
+
+Shutdown describes the configuration for shutting down a server instance.
+
+| Name | Type | Description |
+| --- | --- | --- |
+| ignoreSessions | boolean | For graceful shutdown only, indicates to ignore pending HTTP sessions during in-flight work handling. Not required. Defaults to false. |
+| shutdownType | string | Tells the operator how to shutdown server instances. Not required. Defaults to graceful shutdown. |
+| timeoutSeconds | number | For graceful shutdown only, number of seconds to wait before aborting in-flight work and shutting down the server. Not required. Defaults to 30 seconds. |
```

Comment:
This documentation contradicts documentation elsewhere in this check-in.  Here it's written that timeoutSeconds controls the time until in-flight work is quiesced during graceful shutdown, elsewhere it's documented that it controls when a graceful/forced shutdown is aborted via a kill-9.  

Is an additional configurable needed?